packages/wordcount/README.md

```diff
@@ -0,0 +1,17 @@
+# @wordpress/wordcount
+
+A utility to count words
+
+## Installation
+
+Install the module
+
+```bash
+npm install @wordpress/wordcount@next --save
```

Comment:
I believe we can remove `@next` since we'll probably release a stable version post merge.